server_installation/topics/operating-mode/crossdc/proc-configuring-infinispan.adoc

```diff
@@ -0,0 +1,137 @@
+<3> Sets 1000 as the number of relay nodes for the cluster. You should set a value that is equal to or greater than the maximum number of nodes in your {jdgserver_name} cluster.
+<4> Names all {jdgserver_name} clusters that backup caches with {jdgserver_name} data and uses the default TCP stack for inter-cluster transport.
++
+.. Configure the {jdgserver_name} cluster transport to use the stack.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<cache-container name="default" statistics="true">
+      <transport cluster="${infinispan.cluster.name:cluster}"
+                 stack="xsite"/> <1>
+</cache-container>
+----
+<1> Uses the `xsite` stack for the cluster.
++
+. Configure the keystore as an SSL identity in the server security realm.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<server-identities>
+  <ssl>
+    <keystore path="server.jks" <1>
+              relative-to="infinispan.server.config.path"
+              keystore-password="password" <2>
+              alias="server" /> <3>
+  </ssl>
+</server-identities>
+----
+<1> Specifies the path of the keystore that contains the SSL identity.
+<2> Specifies the password to access the keystore.
+<3> Names the alias of the certificate in the keystore.
++
+. Configure the authentication mechanism for the Hot Rod endpoint.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<endpoints socket-binding="default">
+   <hotrod-connector name="hotrod">
+      <authentication>
+         <sasl mechanisms="SCRAM-SHA-512" <1>
+               server-name="infinispan" /> <2>
+      </authentication>
+   </hotrod-connector>
+   <rest-connector name="rest"/>
+</endpoints>
+----
+<1> Configures the SASL authentication mechanism for the Hot Rod endpoint.  SCRAM-SHA-512 is the default SASL mechanism for Hot Rod. However you can use whatever is appropriate for your environment, such as GSSAPI.
+<2> Defines the name that {jdgserver_name} servers present to clients. You specify this name in the Hot Rod client configuration when you set up {project_name}.
++
+. Create a cache template.
++
+NOTE: Add the cache template to `infinispan.xml` on each node in the {jdgserver_name} cluster.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<cache-container ... >
+  <replicated-cache-configuration name="sessions-cfg" <1>
+                                  mode="SYNC"> <2>
+    <locking acquire-timeout="0" /> <3>
+    <backups>
+      <backup site="site2" strategy="SYNC" /> <4>
+    </backups>
+  </replicated-cache-configuration>
+</cache-container>
+----
+<1> Creates a cache template named `sessions-cfg`.
+<2> Defines a cache that synchronously replicates data across the cluster.
+<3> Disables timeout for lock acquisition.
+<4> Names the backup site for the {jdgserver_name} cluster you are configuring.
++
+. Start {jdgserver_name} at each site.
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ bin/server.sh
```

Comment:
@andymunro FYI, there are some questions about this server start section that @tkyjovsk is looking into.

Reply:
Start server1. 

----
./server.sh -c infinispan.xml -b server1 -Djgroups.mcast_addr=228.6.7.10 
---- 

Start server 2

----
./server.sh -c infinispan.xml -b server2 -Djgroups.mcast_addr=228.6.7.11
----

Reply:
I think we should use `-b PUBLIC_IP_ADDRESS` instead `-b serverN` to be consistent with the rest of the docs.

We should also add the `-k PUBLIC_IP_ADDRESS` parameter here (or alternatively `-Djgroups.bind.address=PUBLIC_IP_ADDRESS` which should be equivalent). Without this the TCPPING for establishing the bridge/relay cluster doesn't work. At least it didn't in my testing.